### PR TITLE
Added Changelog to Aleo Documentation

### PR DESCRIPTION
**Pull request by moAlobaidi**
This PR addresses this Issue #213 . A Changelog has been added to ensure that our documentation remains up-to-date and accessible for our users.

A Changelog is a vital part of any project's documentation. It provides an organized, chronological arrangement of notable changes for each version of the project. This includes new features, bug fixes, and improvements. It allows users, especially developers, to track progress, understand new features, and troubleshoot issues more effectively.

In the context of Aleo, the Changelog will enhance our documentation by providing a quick reference for all substantial updates. This will make it easier for developers to understand the evolution of Aleo, its features, and improvements. It will also allow them to keep track of changes that could potentially impact their work, thereby improving their experience with Aleo.

It is in tandem with this PR https://github.com/AleoHQ/aleo-docs/pull/211 .

### CODE REVIEW

File: documentation/00_changelog.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+The Leo language and Aleo SDK are currently in active development. Please monitor their respective repositories for 
+the latest updates:
+[**Leo**]
+(https://github.com/AleoHQ/leo) [**Aleo**]
+(https://github.com/AleoHQ/aleo) .
+:::
+
+## Changes **06.14.2023**
+---
+### Hash Functions
```
Comment (collinc97) [inline documentation/00_changelog.md:17]: Can you specify that these opcodes are for Aleo Instructions and provide links to the full documentation pages?

**Review by collinc97 — CHANGES_REQUESTED** [review]

File: documentation/00_changelog.md (hunk 1 above)
Comment (moAlobaidi) [reply documentation/00_changelog.md:17]: Yes, will do!

**Review by collinc97 — CHANGES_REQUESTED** [review]
For completeness, the following features should be added to the changelog
- [x] Chacha random opcodes in Aleo Instructions https://github.com/AleoHQ/welcome/pull/210
- [ ] Mapping updates to Aleo Instructions and Leo https://github.com/AleoHQ/welcome/pull/209
- [ ] Branching opcodes in Aleo Instructions https://github.com/AleoHQ/welcome/pull/216
- [ ] `block.height` finalize opcode in Aleo Instructions and Leo https://github.com/AleoHQ/welcome/pull/216

**Comment by moAlobaidi** [comment]
> For completeness, the following features should be added to the changelog
> 
> * [ ]  Chacha random opcodes in Aleo Instructions [document new rand.chacha function #210](https://github.com/AleoHQ/welcome/pull/210)
> * [ ]  Mapping updates to Aleo Instructions and Leo [Update aleo instructions mapping docs #209](https://github.com/AleoHQ/welcome/pull/209)
> * [ ]  Branching opcodes in Aleo Instructions [Update documentation post fork #216](https://github.com/AleoHQ/welcome/pull/216)
> * [ ]  `block.height` finalize opcode in Aleo Instructions and Leo [Update documentation post fork #216](https://github.com/AleoHQ/welcome/pull/216)

Should I create an issue for each of the above items so we can review each separately? It might get hectic adding them here as the original intent for this PR was just adding the changelog section. I added changes I had already written prior to prime it so it's not empty.

**Comment by collinc97** [comment]
> > For completeness, the following features should be added to the changelog
> > 
> > * [ ]   Chacha random opcodes in Aleo Instructions [document new rand.chacha function #210](https://github.com/AleoHQ/welcome/pull/210)
> > * [ ]   Mapping updates to Aleo Instructions and Leo [Update aleo instructions mapping docs #209](https://github.com/AleoHQ/welcome/pull/209)
> > * [ ]   Branching opcodes in Aleo Instructions [Update documentation post fork #216](https://github.com/AleoHQ/welcome/pull/216)
> > * [ ]   `block.height` finalize opcode in Aleo Instructions and Leo [Update documentation post fork #216](https://github.com/AleoHQ/welcome/pull/216)
> 
> @collinc97 Should I create an issue for each of the above items so we can review each separately? It might get hectic adding them here as the original intent for this PR was just adding the changelog section.

I think it will be faster if you just add them to the changelog here - no need to create a bunch of other issues to manage, you are the only one committing to this PR @moAlobaidi

**Comment by moAlobaidi** [comment]
Closing this PR as per our discussion about the sustainability of this approach. 🫡

**Pull request closed without merging**